recipes/libdwarf/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8)
+project(conan_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+if (WIN32 AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
This variable was introduced in CMake 3.4, so that should be the minimum version. Which is the minimum CMake version of the library?

Reply:
@danimtb  good to know, I always thought that variable was pretty old, from cmake 2.8. The minimal cmake version in dwarf is 3.2. Do you think we can promote to 3.4 in the cmake wrapper ?

Reply:
I think it is reasonable. If there is any trouble wi will have the conversation here to check. Thanks!